DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -32,6 +28,23 @@ function PlayerAnyEntry(#var(PlayerPawn) p)
 #endif
 }
 
+function PreFirstEntry()
+{
+    local #var(PlayerPawn) p;
+    local string startMapName;
+
+    foreach AllActors(class'#var(PlayerPawn)',p){break;}
```

Comment:
any reason to not use the `player()` function here?

Reply:
Nope.  I think I had cobbled this together from various attempts in static functions, so it probably just snuck under my radar.